cli/src/main/java/org/openjdk/skara/cli/GitJCheck.java

```diff
@@ -172,4 +178,8 @@ public static void main(String[] args) throws Exception {
             }
         }
     }
+
+    private static boolean isURL(String pathOrURL) {
```

Comment:
Nice to add a static method. Very minor nit, I would probably just have went with `String s` here, but I'm fine with `pathOrURL` as well.

Reply:
Yeah, `String s` seems fine. I'll change it.